core/src/main/java/org/mskcc/cbio/portal/util/StructuralVariantUtil.java

```diff
@@ -32,15 +32,22 @@
 
 package org.mskcc.cbio.portal.util;
 
-import java.util.HashMap;
+import java.io.*;
+import java.util.*;
+
+import com.fasterxml.jackson.databind.*;
+import org.mskcc.cbio.maf.NamespaceColumnParser;
 import org.mskcc.cbio.maf.TabDelimitedFileUtil;
 import org.mskcc.cbio.portal.model.StructuralVariant;
+import org.mskcc.cbio.portal.scripts.*;
 
 /**
  * @author ochoaa
  */
 public class StructuralVariantUtil {
     private HashMap<String, Integer> columnIndexMap;
+    private final NamespaceColumnParser namespaceColumnParser;
+    private ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
`final`

Reply:
Fixed in [52b680d](https://github.com/cBioPortal/cbioportal/commit/52b680d6bf158a8f0f4308e4fa385921110aa5b8)